android/.gitignore

```diff
@@ -94,3 +94,8 @@ capacitor-cordova-android-plugins
 
 # Copied web assets
 app/src/main/assets/public
+
+# Generated Config files
+app/src/main/assets/capacitor.config.json
+app/src/main/assets/capacitor.plugins.json
```

Comment:
They are already in the repo and why do we have generated config under `/app/src/...` ?

Reply:
These were generated by Android Studio when I first built the app, I'm not sure they're needed (the `capacitor.config.json` is the same as the one we have in the root directory.